ytools/y2tool/rubocop_yast_style.yml

```diff
@@ -88,3 +88,9 @@ Style/CaseIndentation:
 Style/NegatedIf:
   Enabled: false
 
+# allow more than 10 lines for methods
+Metrics/MethodLength:
+  Max: 50
```

Comment:
I'd use some smaller number as the global default, like `30`. IIRC Rubocop ignores empty lines (white space) and comments, it counts the real code lines only.

We can raise the limit in a specific module if need, the global default should be a bit stricter IMHO :wink: